main.go

```diff
@@ -536,14 +552,26 @@ func serverListen(context *Context) error {
 		return err
 	}
 
+	// Compile the rego policy
+	allowQuery, err := rego.New(
+		rego.Query(*serverAllowQuery),
+		rego.Load([]string{*serverAllowPolicy}, nil),
+		//rego.Module(ct.String("policy"), string(module)),
+	).PrepareForEval(gocontext.TODO())
```

Comment:
This can be `gocontext.Background()`, since we're doing this processing at startup and don't intent to put a deadline/cancel func on it.